ddtrace/llmobs/_llmobs.py

```diff
@@ -604,6 +605,11 @@ def enable(
 
             # override the default _instance with a new tracer
             cls._instance = cls(tracer=_tracer, span_processor=span_processor)
+
+            # Add APM trace filter to drop all APM traces when DD_APM_TRACING_ENABLED is falsy
+            apm_filter = APMTracingEnabledFilter()
+            cls._instance.tracer._span_aggregator.user_processors.append(apm_filter)
```

Comment:
We should add this processor to dd_processors. User_processors can be reset reset when tracer.configure(...) is called by users